packages/core/src/setup.ts

```diff
@@ -226,9 +277,128 @@ export function setup<
     TMeta,
     TConfig
   >;
+
+  assign: typeof assign<
+    TContext,
+    TEvent,
+    unknown,
+    TEvent,
+    ToProvidedActor<TChildrenMap, TActors>
+  >;
+  sendTo: <
+    TParams extends ParameterizedObject['params'] | undefined,
+    TTargetActor extends AnyActorRef,
+    TUsedDelay extends TDelay = never
+  >(
+    to:
+      | TTargetActor
+      | string
+      | ((
+          args: ActionArgs<TContext, TEvent, TEvent>,
+          params: TParams
+        ) => TTargetActor | string),
+    eventOrExpr:
+      | EventFrom<TTargetActor>
+      | SendExpr<
+          TContext,
+          TEvent,
+          TParams,
+          InferEvent<Cast<EventFrom<TTargetActor>, EventObject>>,
+          TEvent
+        >,
+    options?: SendToActionOptions<TContext, TEvent, TParams, TEvent, TUsedDelay>
+  ) => ActionFunction<
+    TContext,
+    TEvent,
+    TEvent,
+    TParams,
+    never,
+    never,
+    never,
+    TDelay,
+    never
+  >;
+  sendParent: <
+    TParams extends ParameterizedObject['params'] | undefined,
+    TSpecificEvent extends TEvent,
+    TUsedDelay extends TDelay = never
+  >(
+    event:
+      | TSpecificEvent
+      | SendExpr<TContext, TEvent, TParams, TSpecificEvent, TEvent>,
+    options?: SendToActionOptions<TContext, TEvent, TParams, TEvent, TUsedDelay>
+  ) => ActionFunction<
+    TContext,
+    TEvent,
+    TSpecificEvent,
+    TParams,
+    never,
+    never,
+    never,
+    TDelay,
+    never
+  >;
+  forwardTo: <TParams extends ParameterizedObject['params'] | undefined>(
```

Comment:
I'd probably remove `forwardTo` and `sendParent` from here. Those have questionable type safety and are mostly wrappers over more type-safe alternatives

Reply:
I removed `forwardTo` but I'm leaving the decision about `sendParent`'s removal up to you.